.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,155 @@
+
+    Returns:
+        bool: True if eslint-disable statement is found, False otherwise.
+    """
+    eslint_disable_pattern = re.compile(
+        r"""\/\/\s*eslint-disable(?:-next-line
+        |-line)?[^\n]*|\/\*\s*eslint-disable[^\*]*\*\/""",
+        re.IGNORECASE,
+    )
+
+    try:
+        with open(file_path, "r", encoding="utf-8") as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)